.github/workflows/deploy.yml

```diff
@@ -28,20 +33,26 @@ jobs:
       id: buildx
       uses: docker/setup-buildx-action@v1
 
-    - name: Build and push base image
+    - name: Build and push base image (${{ matrix.dockerbase }})
       id: docker_build
       uses: docker/build-push-action@v2
       with:
-        context: ./docker/base
+        context: ./docker/base/${{ matrix.dockerbase }}
         push: true
         tags: |
-          exaworks/sdk-base:latest
+          exaworks/sdk-base:${{ matrix.dockerbase }}
         platforms: linux/amd64
 
-    - name: Build and push composite image
+    - name: Build and push composite image (${{ matrix.dockerbase }})
       run: |
-        docker build -t rp docker/rp
+        docker build -t rp docker/rp --build-arg BASE_IMAGE=exaworks/sdk-base:${{ matrix.dockerbase }}
         docker build -t rp_parsl --build-arg BASE_IMAGE=rp docker/Parsl
         docker build -t rp_parsl_swift --build-arg BASE_IMAGE=rp_parsl docker/swift-t
         docker build -t exaworks/sdk --build-arg BASE_IMAGE=rp_parsl_swift docker/flux
-        docker push exaworks/sdk
\ No newline at end of file
+        docker push exaworks/sdk:${{ matrix.dockerbase }}
+
+    - name: Push images with 'latest' tag
+      if: ${{ matrix.dockerbase == 'centos7' }}
```

Comment:
just to confirm, should it be CentOS 7 or 8?

Reply:
I'm not sure.  I don't have any strong preferences one way or another. I honestly just picked one.  Any thoughts?

Reply:
same :) no preferences, ok to keep as it is